http-servers/html-server.js

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs';
+
+const requestHandler = (request, response) => {
+    var filename = __dirname+'/../index.html';
```

Comment:
Please don't resolve the path as a simple string, here's the method for doing that:
https://nodejs.org/dist/latest-v10.x/docs/api/path.html#path_path_resolve_paths